CMSgov/design-system
# [WNMGDS 1166] Add lightweight alert

## Summary
Adding a lightweight alert weight option to our alerts

### Added
- Added a new prop `weight` to the alert with the current only option being `"lightweight"`.
- Added new styling to set the background color to white for the lightweight alerts

### Fixed
List any bug fixes.

## How to test
- Run the doc site locally with `yarn start`
- Add `weight="lightweight"` to an alert component and see that the alert gets the lightweight styling with a color left border and white background
- Demo URL http://design-system-demo.s3-website-us-east-1.amazonaws.com/WNMGDS-1166-lightweight-alert/components/alert/
